src/plugins/kdb/test/kdb_test.c

```diff
@@ -535,7 +579,7 @@ test_check_allowed_to_delegate(krb5_context context,
 
 kdb_vftabl PLUGIN_SYMBOL_NAME(krb5_test, kdb_function_table) = {
     KRB5_KDB_DAL_MAJOR_VERSION,             /* major version number */
-    0,                                      /* minor version number 0 */
+    2,                                      /* minor version number 0 */
```

Comment:
Should be 1, yeah?

Reply:
Right. I was mislead by the comment in kdb.h ("End of minor version 1."), but I think it got decremented when the major version was incremented (see both commits with 'git log -S free_principal_e_data').